tasks/agent.py

```diff
@@ -933,6 +935,30 @@ def omnibus_build(
     with timed(quiet=True) as bundle_elapsed:
         bundle_install_omnibus(ctx, gem_path, env)
 
+    omnibus_cache_dir = os.environ.get('OMNIBUS_GIT_CACHE_DIR')
+    use_omnibus_git_cache = omnibus_cache_dir is not None
+    if use_omnibus_git_cache:
+        omnibus_cache_dir += install_directory
```

Comment:
This seems fragile as a path concatenation operation, should that use [os.path.join](https://docs.python.org/3/library/os.path.html#os.path.join) instead?